src/routes/tools/_common/_constants.js

```diff
@@ -0,0 +1,120 @@
+import models from "~/helpers/climate-models";
```

Comment:
for the files in the `_common` directory I don't think they need an underscore prefix because Sapper won't recognize this directory as a possible route.

Reply:
Fixed via https://github.com/berkeley-gif/caladapt-website-2021/commit/c0a44e0d6aab0fd53771aec531cb872efe8de3bc